Source/Models/FORMTarget.m

```diff
@@ -11,18 +11,47 @@ @implementation FORMTarget
 #pragma mark - Initializers
 
 - (instancetype)initWithDictionary:(NSDictionary *)dictionary {
-    self = [super init];
+    self = [super initWithDictionary:dictionary];
     if (!self) return nil;
 
     _targetID = [dictionary andy_valueForKey:@"id"];
     self.typeString = [dictionary andy_valueForKey:@"type"];
     self.actionTypeString = [dictionary andy_valueForKey:@"action"];
-    self.targetValue = [dictionary andy_valueForKey:@"target_value"];
+    self.value = [dictionary andy_valueForKey:@"value"];
```

Comment:
Can we add backwards compatibility? Might be nice if we don't have to jump to Form 4.0.0